source/MRViewer/ImGuiMenu.cpp

```diff
@@ -385,4 +423,168 @@ ImGuiContext* ImGuiMenu::getCurrentContext() const
     return context_;
 }
 
+void ImGuiMenu::draw_helpers()
+{
+    if ( ImGui::GetIO().KeysDown[GLFW_KEY_F1] )
```

Comment:
Please put this under flag, that will be controlled from RibbonMenu (F1 reaction should be specified in RibbonMenu)